src/abstractions/serialization/KiotaJsonSerializer.Serialization.cs

```diff
@@ -20,9 +20,21 @@ public static partial class KiotaJsonSerializer
     /// </summary>
     /// <typeparam name="T">Type of the object to serialize</typeparam>
     /// <param name="value">The object to serialize.</param>
+    /// <param name="useBackingStore">Backing store keeps track of changes, setting this to false will give you the full item.</param>
     /// <returns>The serialized representation as a stream.</returns>
-    public static Stream SerializeAsStream<T>(T value) where T : IParsable
-    => KiotaSerializer.SerializeAsStream(_jsonContentType, value);
+    public static Stream SerializeAsStream<T>(T value, bool useBackingStore = true) where T : IParsable
+    => KiotaSerializer.SerializeAsStream(_jsonContentType, value, useBackingStore);
+
+    /// <summary>
+    /// Serializes the specified object as a string based JSON stream.
+    /// </summary>
+    /// <typeparam name="T">The type of the value to serialize.</typeparam>
+    /// <param name="value">The object to serialize.</param>
+    /// <param name="useBackingStore">A flag indicating whether to use a backing store for serialization.</param>
+    /// <returns>A <see cref="Stream"/> containing the serialized JSON data.</returns>
+
+    public static Stream SerializeAsJsonStream<T>(this T value, bool useBackingStore = true) where T : IParsable
```

Comment:
we need to replicate the change for all other public methods (collections, etc...)

Reply:
You mean you want the extension method also on collections?
And maybe also on the string versions?

Reply:
I meant the optional parameter